subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/internal/finalizer/verdict/TestStatisticsCounter.kt

```diff
@@ -0,0 +1,16 @@
+package com.avito.instrumentation.internal.finalizer.verdict
+
+internal interface TestStatisticsCounter {
+
+    fun overallDuration(): Int
```

Comment:
`Int` doesn't tell about unit of measurement.
Let's add at least prefix to method `overallDurationSec()`